lang/fr.xml

```diff
@@ -183,6 +183,7 @@ Ne pas éteindre le système, ni fermer l'application.</warning_saving>
 		<notif_deleted>Les notifications vont être supprimées.</notif_deleted>
 		<install_failed>L'installation a échoué.</install_failed>
 		<install_complete>Installation terminée.</install_complete>
+		<installin>Installation en cours</installin>
```

Comment:
Why you duplicate ?

Reply:
I just want to separate, if want to unify i can be restored.

Reply:
i no sur psvita have this string 🤔

Reply:
> i no sur psvita have this string 🤔

i restore it.